Tests fail with 1.11.2

After upgrading to 1.11.2, my tests using pytest-mock start failing with the message below. It turns out [this change](https://github.com/pytest-dev/pytest-mock/commit/845876e95677f9fdb906d9a277829a8abcad39da#diff-bbf549d84d8938d3f816f5373049ed89R160) fixes it but it still needs to be released.

I am using pytest 5.2.4

```
self = <pytest_mock.plugin.MockFixture._Patcher object at 0x104e8c590>
stack = [FrameInfo(frame=<frame at 0x104cfc5c0, file '/Users/powerjo/.local/share/virtualenvs/backend-qQJPOvQO/lib/python3.7/s...92, function='_hookexec', code_context=['        return self._inner_hookexec(hook, methods, kwargs)\n'], index=0), ...]

    def _enforce_no_with_context(self, stack):
        """raises a ValueError if mocker is used in a with context"""
        caller = stack[1]
        frame = caller[0]
        info = inspect.getframeinfo(frame)
>       code_context = " ".join(info.code_context).strip()
E       TypeError: can only join an iterable

../../../../.local/share/virtualenvs/backend-qQJPOvQO/lib/python3.7/site-packages/pytest_mock/plugin.py:167: TypeError
```

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -19,4 +19,3 @@ repos:
         files: ^(CHANGELOG.rst|README.rst|HOWTORELEASE.rst|changelog/.*)$
         language: python
         additional_dependencies: [pygments, restructuredtext_lint]
-        python_version: python3.6

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -1,3 +1,12 @@
+1.12.0 (2019-11-19)
+-------------------
+
+* Now all patch functions also raise a ``ValueError`` when used
+  as a context-manager. Thanks `@AlexGascon`_ for the PR (`#168`_).
+
+.. _@AlexGascon: https://github.com/AlexGascon
+.. _#168: https://github.com/pytest-dev/pytest-mock/pull/168
+
 1.11.2 (2019-10-19)
 -------------------
 

